### ISSUE TITLE
[Table]: Accessibility: Row selection using keyboard, does not work correctly.

### ISSUE THREAD

**Issue by stuckless** [IBM/carbon-addons-iot-react#402]
**Steps**
1. Open storybook, navigate to stateful table
2. Click on first row select box (that works)
3. Tab to next row (second row selection should be focussed - blue box)
4. Hit "space" - Selection count updates, but, the select box doesn't update

**Expected results**
Hitting space on the row select box should update the table selection (it does) AND update the UI to reflect the row is selected.

This was tested on the latest carbon 10 storybook.
https://beta.carbon-addons-iot-react.com/?path=/story/table--simple-stateful-example

**Additional notes
Carbon's DataTable ha**ndles this correctly.

**Other UI notes**
the clicks do appear to keep focus of elements in the table.  ie, clicking the expand collapse doesn't keep the blue focus around the chevron.  Same is true for row selection.  It doesn't keep the blue box around the focussed item.

**Comment by tay1orjones** [comment]
Completed in #703
